specification/syntex/resource-manager/Microsoft.Syntex/preview/2022-09-15-preview/examples/DocumentProcessor_List.json

```diff
@@ -0,0 +1,46 @@
+{
+  "parameters": {
+    "subscriptionId": "cd0d3cc8-cae2-4559-b882-cd707170799b",
+    "api-version": "2022-09-15-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/cd0d3cc8-cae2-4559-b882-cd707170799b/resourceGroups/mygroup/providers/microsoft.syntex/documentProcessors/myprocessor1",
```

Comment:
>microsoft.syntex [](http://example.com/codeflow?start=104&length=16)

I think the consistent capitalization of provider names is preferable in resource ids too - even though they are treated case insensitively.